apps/rehabstod/src/pages/CurrentSickLeaves/sickLeaveSlice.ts

```diff
@@ -24,6 +28,7 @@ const sickLeaveSlice = createSlice({
     },
     updateFilter(state, { payload }: PayloadAction<Partial<SickLeaveFilter>>) {
       Object.assign(state.filter, payload)
+      state.hasAppliedFilters = !_.isEqual(initialState.filter, state.filter)
```

Comment:
We would need to install `lodash` for this function, or we could use JSON.stringify() to compare them.